docs/ingestion/dynamodb.md

```diff
@@ -0,0 +1,56 @@
+# DynamoDB
+Amazon [DynamoDB](https://aws.amazon.com/dynamodb/) is a managed NoSQL database service provided by Amazon Web Services (AWS). It supports key-value and document data structures and is designed to handle a wide range of applications requiring scalability and performance. 
+
+
+Bruin supports DynamoDB as a source for [Ingestr assets](/assets/ingestr), and you can use it to ingest data from DynamoDB into your data warehouse.
+
+In order to set up DynamoDB connection, you need to add a configuration item in the `.bruin.yml` file and in `asset` file. 
+
+Follow the steps below to correctly set up DynamoDB as a data source and run ingestion:
+
+### Step 1: Add a connection to .bruin.yml file
+
+To connect to DynamoDB, you need to add a configuration item to the connections section of the `.bruin.yml` file. This configuration must comply with the following schema:
+
+```yaml
+    connections:
+      dynamodb:
+        - name: "my-dynamodb"
+          access_key_id: "AWS_ACCESS_KEY_ID"
+          secret_access_key: "AWS_SECRET_ACCESS_KEY"
+          region: "AWS_REGION"
+          
+```
+
+* `access_key_id`: Identifes an IAM account.
+* `access_token`: Password for the IAM account.
```

Comment:
```suggestion
* `secret_access_key`: Password for the IAM account.
```

not sure exactly how it was but can you please make sure these credential field names match the same for other AWS-related connections such as Athena?